themes/bootstrap3/theme.config.php

```diff
@@ -42,6 +42,7 @@
         ['file' => 'vendor/validator.min.js', 'priority' => 140],
         ['file' => 'lib/form-attr-polyfill.js', 'priority' => 210], // input[form] polyfill, cannot load conditionally, since we need all versions of IE
         ['file' => 'lib/autocomplete.js', 'priority' => 220],
+        ['file' => 'lib/ajax_request_queue.js', 'priority' => 230],
```

Comment:
@crhallberg, since we're emptying out the lib directory in #2755, should we put this new file somewhere else? Or alternatively, should we just make sure that proper style checking is getting applied to the lib directory, since I believe it was being skipped/ignored in the past?

Reply:
I just took a quick look at eslint settings in build.xml, and I'm not seeing any explicit exclusion of either the lib or vendor subdirectories. This makes me wonder if perhaps eslint is operating in a non-recursive mode, and only checking files in the root of the js directory. Do we need to adjust the command and/or .eslintrc file to make things smarter?

Reply:
Since this particular file has a utility that doesn't do anything when loaded (as compared to every other file), I think it is the only candidate for this folder. Then again, if it is the only one, we don't need the folder.

I'll look into the eslint config though.

Reply:
I think we could fix this by [changing the blobs in build.xml](https://github.com/vufind-org/vufind/blob/dev/build.xml#L179-L188), or we could make build.xml more concise by using the [`files` and `ignore` properties](https://eslint.org/docs/latest/use/configure/configuration-files-new#configuration-objects) in `.eslintrc.js`.

Reply:
I would favor moving the configuration to .eslintrc.js, since I think that makes things a little easier to manage. Do you want to address that through a separate PR?